pegasus/sites.v3/code.org/public/accessibility.haml

```diff
@@ -0,0 +1,100 @@
+---
+title: 'Accessibility'
+theme: responsive_full_width
+---
+
+%link{href: "/css/generated/design-system-pegasus.css", rel: "stylesheet"}
+%link{href: "/css/generated/page/accessibility-styles.css", rel: "stylesheet"}
+
+%section.hero-banner-basic.hero-banner-basic--accessibility
+  .wrapper
+    .flex-container
+      .text-wrapper.col-50.flex-container.align-items-center
+        .hero-banner-basic__content
+          %h1{style: "color: white"}
+            =hoc_s(:accessibility_hero_heading)
+          %p{style: "color: white"}
+            =hoc_s(:accessibility_hero_desc)
+      .col-50.flex-container.justify-end
+        %img{src: "/images/accessibility/hero-image.png", alt: ""}
+    .clear
+
+%section.accessibility-efforts.bg-neutral-light
+  .wrapper
+    %h2
+      = hoc_s(:accessibility_efforts_title)
+    %p
+      = hoc_s(:accessibility_efforts_desc)
+    .grid-container
+      - item_count = 4
+      - item_count.times do |index|
+        .flex-container
+          %figure
+            %img{src: "/images/accessibility/efforts_#{index + 1}.png",  alt: hoc_s("accessibility_efforts_card_title_#{index + 1}")}
+          .text-wrapper
+            %h3.heading-sm
+              =hoc_s("accessibility_efforts_card_title_#{index + 1}")
+            %p
+              - if index == 1
+                = hoc_s(:accessibility_efforts_card_desc_2_a)
+                %a{href: "https://www.section508.gov/sell/vpat/", target: "_blank", rel: "noopener noreferrer"}
+                  = hoc_s(:accessibility_efforts_card_desc_2_b)
+                = hoc_s(:accessibility_efforts_card_desc_2_c)
+              - elsif index == 3
+                = hoc_s(:accessibility_efforts_card_desc_4_a)
+                %a{href: "https://www.w3.org/TR/WCAG20/", target: "_blank"}
+                  = hoc_s(:accessibility_efforts_card_desc_4_b)
+                = hoc_s(:accessibility_efforts_card_desc_4_c)
+              - else
+                = hoc_s("accessibility_efforts_card_desc_#{index + 1}")
+
+%section.accessibility-explore
+  .wrapper
+    %h2=hoc_s(:accessibility_explore_title)
+    .action-block.action-block--two-col
+      %img{src: "/images/accessibility/explore.jpg", alt: hoc_s(:accessibility_explore_title), class: "col-50"}
+      .text-wrapper.col-50
+        %p.heading-sm=hoc_s(:accessibility_explore_text)
+        %a{href: "https://support.code.org/hc/en-us/sections/15186591538957-Accessibility", class: "link-button", target: "_blank"}
```

Comment:
For security reasons can you make sure to include `rel: "noopener noreferrer"` on all links that open in a new tab?

Reply:
done